crates/holochain_chc/src/chc_http.rs

```diff
@@ -1,22 +1,78 @@
 //! Defines a client for use with a remote HTTP-based CHC.
+//!
+//! The server must implement the following endpoints:
+//!
+//! ## `POST /add_records/{dna_hash}/{agent_pubkey}`
```

Comment:
The docs suggest that the CHC http server should accept the `dna_hash` and `agent_pubkey` path params, but in the actual implementation, these path params are not passed

https://github.com/holochain/holochain/blob/60102a603b8039eb46e786d82dd6382f3a1b1c93/crates/holochain/src/conductor/chc/chc_remote.rs#L114

Reply:
Ahh I see, thanks for the clarification